src/Orleans/Messaging/GatewayConnection.cs

```diff
@@ -66,7 +67,6 @@ public override void Stop()
             IsLive = false;
             receiver.Stop();
             base.Stop();
-            RuntimeClient.Current.BreakOutstandingMessagesToDeadSilo(Silo);
```

Comment:
lets please not remove that for now. I am still not convinced that is the right things to do, and as we said: this is unrelated to saving the msgs in the queue.
Once the queue resend is submitted, you can open a new PR with this line removed only and test and we can discuss if this is needed or not.

I am strongly opposing on bundling independent fixes/changes.